src/mmw/js/src/compare/tests.js

```diff
@@ -0,0 +1,235 @@
+            });
+
+            describe('#update', function() {
+                it('uses precolumbian results for 100% forest cover scenario',
+                   function() {
+                       var precolumbianResult = this.scenarios
+                           .findWhere({ is_pre_columbian: true })
+                           .get('results')
+                           .findWhere({ name: 'runoff' })
+                           .get('result'),
+                           precolumbianValue = precolumbianResult.runoff['pc_unmodified'];
+
+                       this.chartRows.update();
+
+                       this.chartRows.forEach(function(chart) {
+                           var forestValue = chart.get('values')[0];
+                           if (chart.get('key') === 'combined') {
+                               assert.equal(forestValue, precolumbianValue);
+                           } else {
+                               assert.equal(forestValue,
+                                            precolumbianValue[chart.get('key')]);
+                           }
+                       });
+                   });
+
+                it('uses unmodified results for current conditions scenario',
+                   function() {
+                       var unmodifiedResult = this.scenarios
+                           .findWhere({ is_current_conditions: true })
+                           .get('results')
+                           .findWhere({ name: 'runoff' })
+                           .get('result'),
+                           unmodifiedValue = unmodifiedResult.runoff['unmodified'];
+
+                       this.chartRows.update();
+
+                       this.chartRows.forEach(function(chart) {
+                           var currentConditionsValue = chart.get('values')[1];
+                           if (chart.get('key') === 'combined') {
+                               assert.equal(currentConditionsValue,
+                                            unmodifiedValue);
+                           } else {
+                               assert.equal(currentConditionsValue,
+                                            unmodifiedValue[chart.get('key')]);
+                           }
+                       });
+                   });
+
+                it('uses modified results for user-created scenario',
+                   function() {
+                       var isUserCreated = function(scenario) {
+                           return !scenario.get('is_current_conditions') &&
+                               !scenario.get('is_pre_columbian');
+                       },
+
+                           modifiedResult = this.scenarios
+                           .find(isUserCreated)
+                           .get('results')
+                           .findWhere({ name: 'runoff' })
+                           .get('result'),
+                           modifiedValue = modifiedResult.runoff['modified'];
```

Comment:
Consider indenting these (and other) `.get` &c calls to distinguish them visually from the variables here.

Reply:
Someday I will correct my editor's auto-indent...will definitely make this look right before merging.